test/e2e_node/node_shutdown_linux_test.go

```diff
@@ -57,6 +61,27 @@ var _ = SIGDescribe("GracefulNodeShutdown [Serial] [NodeFeature:GracefulNodeShut
 	f := framework.NewDefaultFramework("graceful-node-shutdown")
 	f.NamespacePodSecurityLevel = admissionapi.LevelPrivileged
 
+	ginkgo.BeforeEach(func() {
+		systemdVersion := 0
+		if _, err := exec.LookPath("systemd-run"); err == nil {
+			if version, verr := exec.Command("systemd-run", "--version").Output(); verr == nil {
+				// sample output from $ systemd-run --version
+				// systemd 245 (245.4-4ubuntu3.13)
+				re := regexp.MustCompile(`systemd (\d+)`)
+				if match := re.FindSubmatch(version); len(match) > 1 {
+					systemdVersion, _ = strconv.Atoi(string(match[1]))
```

Comment:
need some logging in case we failed to parse the version so it is easier to troubleshoot why something is skipped

Reply:
ack will add more